### PR TITLE
Issue 150

### PR DESCRIPTION
**Pull request by williamcroberts**

### CODE REVIEW

**Comment by flihp** [comment]
Says travis build failed but I can't find the failure in the logs. Are my eyes broken today?

**Comment by williamcroberts** [comment]
Session mess with --without-tcti-socket... its 100% broken and I'm now dealing with how to pull this apart...

**Comment by williamcroberts** [comment]
Happiest with this version....

File: src/tpm_hmac.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+ * tpm_hmac.h
+ *
+ *  Created on: Feb 1, 2017
+ *      Author: wcrobert
+ */
+
```
Comment (williamcroberts) [inline src/tpm_hmac.h]: dang thought I got all these

Comment (williamcroberts) [inline src/tpm_hmac.h]: needs copyright notice

File: src/tpm_hmac.c [hunk 2]
```diff
@@ -0,0 +1,121 @@
+#include <stdbool.h>
```
Comment (williamcroberts) [inline src/tpm_hmac.c]: needs copyright

**Pull request merged**